tests/php/test-amp-helper-functions.php

```diff
@@ -225,6 +226,7 @@ function ( $old_version ) use ( $options ) {
 		add_action( 'after_setup_theme', 'amp_after_setup_theme', 5 );
 		do_action( 'after_setup_theme' );
 		do_action( 'init' );
+		do_action( 'admin_init' );
```

Comment:
As mentioned in the previous comment, should the `admin_init` replace the `init` action here, or should they both be used?